### PR TITLE
Fix fv for policy sync with binder.

### PR DESCRIPTION
**Pull request by saumoh**
## Description
* Added the references to the binder code in the FV for policy sync.
* Removed Felix configuration path knob that is no longer needed with the binder.

### CODE REVIEW

File: felix.go [hunk 1]
```diff
@@ -28,14 +28,14 @@ import (
...
 
-	"github.com/docopt/docopt-go"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 	log "github.com/sirupsen/logrus"
 	"k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
 
+	"github.com/docopt/docopt-go"
```
Comment (spikecurtis) [inline felix.go]: looks like this was moved from the previous block, but it should stay there.

File: felix.go [hunk 2]
```diff
@@ -315,8 +315,8 @@ configRetry:
 	var policySyncProcessor *policysync.Processor
 	var policySyncAPIBinder binder.Binder
 	calcGraphClientChannels := []chan<- interface{}{dpConnector.ToDataplane}
-	if configParams.PolicySyncManagementSocketPath != "" {
-		log.WithField("managementSocket", configParams.PolicySyncManagementSocketPath).Info(
+	if configParams.PolicySyncPathPrefix != "" {
+		log.WithField("policy sync path prefix ", configParams.PolicySyncPathPrefix).Info(
```
Comment (spikecurtis) [inline felix.go]: I think the style here is to use camelCase for logging fields

File: felix.go [hunk 3]
```diff
@@ -470,7 +470,7 @@ configRetry:
 	dpConnector.Start()
 
 	if policySyncProcessor != nil {
-		log.WithField("managementSocket", configParams.PolicySyncManagementSocketPath).Info(
+		log.WithField("policy sync directory", configParams.PolicySyncPathPrefix).Info(
```
Comment (spikecurtis) [inline felix.go]: camelCase, and this should be the same field name as the above log statement, for easier grepping of logs

File: fv/policysync_test.go [hunk 4]
```diff
@@ -66,8 +67,7 @@ var _ = Context("policy sync API tests", func() {
 
 		// Configure felix to enable the policy sync API.
 		options := containers.DefaultTopologyOptions()
-		options.ExtraEnvVars["FELIX_PolicySyncManagementSocketPath"] = "/var/run/calico/policy-mgmt.sock"
-		options.ExtraEnvVars["FELIX_PolicySyncWorkloadSocketPathPrefix"] = "/var/run/calico"
+		options.ExtraEnvVars["FELIX_PolicySyncPathPrefix"] = "/var/run/calico"
```
Comment (spikecurtis) [inline fv/policysync_test.go:70]: Comment above still refers to the management socket

**Review by spikecurtis — APPROVED** [review]
A few nits inline, but this looks good

**Comment by saumoh** [comment]
@spikecurtis Thanks for the review. I've incorporated the changes.

**Pull request merged**